tests/e2e-playwright/tests/sim4life.py

```diff
@@ -42,7 +46,7 @@ def test_sim4life(
     _textbox.press("Enter")
     page.get_by_test_id(service_test_id).click()
 
-    with page.expect_response(re.compile(r"/projects/")) as response_info:
+    with page.expect_response(projects_uuid_pattern) as response_info:
```

Comment:
maybe think of re-using:
create_new_project_and_delete
available in conftests.py

Reply:
see in sleepers.py for an example